_data/students.yml

```diff
@@ -53,7 +53,14 @@
   github: Jeewal
   image: JK.jpg
   twitter: jeewalkunwar1
-  
+
 - name: Filip Michalak
   github: Rekjef
   image: RekjefAvatar.png
+
+- name: Maciej Synowski
+  github: Shizerq
+  image: Shizerq.jpg
+  twitter: maciej_synowski
+=======
+  >>>>>>> master
```

Comment:
remove it too.

Reply:
It stayed accidentally, while i was fixing conflict. Deleted.